### PR TITLE
hammerable glass shards (for sand)

### PR DESCRIPTION
**Pull request by jjpark-kb**
## About The Pull Request
pound... sand?
You can now use any tool that has the hammer behavior on it (xenoarch hammer, forging hammer) to break shards back into sand.

## How This Contributes To The Skyrat Roleplay Experience
sand is really just fine pieces of glass, shells, etc. so this makes sense in that way. Also, if you accidentally smelt a piece of sand rather than turning it into clay for ceramics, this allows you to at least undo that mistake.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
image

</details>

## Changelog

:cl:
add: you can hammer glass shards to get back sand
/:cl:

### CODE REVIEW

**Comment by Elli-Skala** [comment]
Infinite mining point exploit

**Comment by jjpark-kb** [comment]
> Infinite mining point exploit

Not the best way to give feedback, but addressed.

File: modular_skyrat/modules/primitive_production/code/misc.dm [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/obj/item/shard/attackby(obj/item/item, mob/user, params)
+	//xenoarch hammer, forging hammer, etc.
+	if(item.tool_behaviour == TOOL_HAMMER)
+		new /obj/effect/decal/cleanable/glass(get_turf(src))
+		new /obj/item/stack/ore/glass/zero_cost(get_turf(src))
+		user.balloon_alert(user, "[src] shatters!")
+		playsound(src, SFX_SHATTER, 30, TRUE)
+		qdel(src)
+		return
```
Comment (vinylspiders) [inline modular_skyrat/modules/primitive_production/code/misc.dm]:
```suggestion
		return TRUE
```

**Review by vinylspiders — CHANGES_REQUESTED** [review]

**Review by vinylspiders — APPROVED** [review]
Might also want to add a desc to indicate the sand produced this way is worthless/ gives no mining points

**Review by vinylspiders — APPROVED** [review]

**Pull request merged**